docs/platform/local-development/set-up-frontend-environment.md

```diff
@@ -1,46 +1,75 @@
 # Setting up your local develop environment
+
 If you want to contribute to the LITTIL website you need to set up a development environment. The website static environment based on [Angular.io](https://angular.io).
 
 ## Prerequisites
+
 - NodeJS LTS >= 16
 - NPM 8
 
 ### Nodejs
+
 Nodejs is used to generate the static files via Angular. The LITTIL website is tested and build on the LTS version of NodeJS 16.
-When installing NodeJS you will get an installation of NPM as well. 
+When installing NodeJS you will get an installation of NPM as well.
 
 #### Windows & MacOS
+
 To install NodeJS on Windows or MacOS you can simply download it from [the website](https://nodejs.org/en/).
 
 #### Linux
+
 The easiest way to install NodeJS is by using your package manager. For instructions on your distribution see [the website as well](https://nodejs.org/en/download/package-manager/)
 
 ### Installation
-To download and install all your dependencies you can run:
+
+To run the Frontend locally, follow these steps:
+
+1. Download the repository from [Github|https://github.com/Devoxx4Kids-NPO/littil-frontend]
+1. Open the repository folder in your IDE ([VSCode download page|https://code.visualstudio.com/download])
+1. Open a terminal and run:
 
 ```shell
 $ npm install
 ```
 
+1. To get the OpenAPI specs locally, run:
+
+```shell
+$ npm openapi
```

Comment:
```npm run api``` is het command toch?

Reply:
Als je in de package.json bij scripts  kijkt, zie je het command `openapi` staan ;)